### DISCUSSION TITLE
About rke2 certificate rotate-ca and K10 token forma

### DISCUSSION THREAD

**Discussion by leagueofsoups** [rancher/rke2#4316] [Q&A] [answered]
Can anybody explain situation with CA rotate and K10 token format?
Should we expect influence on service access after CA rotate?
Do I understand correctly that after rotating the CA, the token will need to be updated on other nodes and restart the server/agent service?

**Comment by brandond** [comment, marked as answer]
Please check out the docs at:
* https://docs.k3s.io/cli/certificate
* https://docs.k3s.io/cli/token

This same content will eventually be ported over to rke2 almost verbatim; you just need to replace any k3s references with rke2.

**Reply by leagueofsoups** [reply]
thx
